analysis/runtime/src/events.rs

```diff
@@ -90,8 +101,10 @@ impl Debug for EventKind {
             } => write!(f, "realloc(0x{:x}, {}) -> 0x{:x}", old_ptr, size, new_ptr),
             Ret(ptr) => write!(f, "ret(0x{:x})", ptr),
             Done => write!(f, "done"),
+            BeginBody => write!(f, "begin body"),
             LoadAddr(ptr) => write!(f, "load(0x{:x})", ptr),
             StoreAddr(ptr) => write!(f, "store(0x{:x})", ptr),
+            StoreAddrTaken(ptr) => write!(f, "store(0x{:x})", ptr),
```

Comment:
Should these print the same?  I would think we'd want to distinguish them.

Reply:
We only want to distinguish them to treat of kinds of instrumentation points differently, this is treated as a store in the same way `StoreAddr` is